samcli/commands/local/lib/local_lambda.py

```diff
@@ -30,6 +33,13 @@
 
 LOG = logging.getLogger(__name__)
 
+RUST_LOCAL_INVOKE_DISCLAIMER = """
+DISCLAIMER: If your local machine is using apple silicon and you are unable to run \"sam local invoke\" 
```

Comment:
nit - (capital A for Apple)
```suggestion
DISCLAIMER: If your local machine is using Apple silicon and you are unable to run \"sam local invoke\" 
```